internal/vm/hcs/pci.go

```diff
@@ -21,6 +21,7 @@ func (uvm *utilityVM) AddDevice(ctx context.Context, instanceID, vmbusGUID strin
 					DeviceInstancePath: instanceID,
 				},
 			},
+			PropagateNumaAffinity: true,
```

Comment:
This should be behind an OS build check. Can you (for now) put WS2025 build? And have a TODO to update to actual build number in the future?
To be safe, we could make the field be pointer to `bool` and default to `nil` value. This way we'll be able to: skip the field for older builds and pass true/false for newer builds. For example, you can check `HclEnabled` field in `/internal/hcs/schema2/isolation_settings.go` and how it's set in `create_lcow.go` and `oci/uvm.go`